docs/book/src/topics/managedcluster.md

```diff
@@ -317,6 +317,27 @@ spec:
   osDiskType: "Ephemeral"
 ```
 
+### AKS Node Pool Taints
+
+You can configure the `Taints` value for each AKS node pool (`AzureManagedMachinePool`) that you define in your spec.
```

Comment:
We haven't been doing this for other fields but should we point out in docs when fields are immutable?

Reply:
yes! I will add, as a user, it would be really helpful. I will do a repass and add to other fields for managedmachinepool also